packages/cozy-stack-client/src/NotesCollection.js

```diff
@@ -47,6 +53,26 @@ class NotesCollection extends DocumentCollection {
       data: { ...normalizeNote(resp.data), _deleted: true }
     }
   }
+
+  /**
+   * open - Returns the details to build the note's url
+   *
+   * @see https://github.com/cozy/cozy-stack/blob/master/docs/notes.md#get-notesidopen
+   *
+   * @param {io.cozy.notes} note The note document to open
+   * @param {string}   note._id The note's id
+   *
+   * @returns {{ data }} The note's url details
+   */
+  async open({ _id }) {
```

Comment:
When I read the description, I find that `fetchDetails` or `fetchURL` would be more appropriate ?

Reply:
Sounds good to me.